jobs/nessus-agent/templates/config/crontab

```diff
@@ -0,0 +1,3 @@
+# m h dom mon dow user  command
+# Sleep a random amount of time from 0-5 minutes to avoid all agents relinking at the same time
+30 * * * * root /bin/sleep $(( ( RANDOM % 300 )  + 1 ))  && /var/vcap/jobs/nessus-agent/bin/link-agent.sh
```

Comment:
Is this supposed to run every 30 minutes? Does it matter if the agent is already linked?